src/View/mainpage/MenuBar.py

```diff
@@ -1,528 +1,46 @@
-import sys
-from copy import deepcopy
+import webbrowser
 
-from PyQt5 import QtCore, QtGui, QtWidgets
-from PyQt5.QtWidgets import QMessageBox
+from PyQt5 import QtWidgets, QtCore, QtGui
 
-from src.Model.CalculateImages import *
+from src.Controller.ActionHandler import ActionHandler
```

Comment:
presumably most of what had been in this module got factored out to ActionHandler?